Architecture.md

```diff
@@ -0,0 +1,100 @@
+# Architecture
+
+## High Level
+
+| Entry Point | Game Loop | Outputs        |
+|-------------|-----------|--------------: |
+| main.js     | game.js   | SIG_TERM, Save |
+
+### `main.js`
+
+Serves as a boostrapping file for the environment the game will run in.
+
+Provides to `game.js`:
+
+- EventBus -- backbone of the physical game loop. Must be synchronous.
+- GameState -- a clean one for a new game, else a dirty one for a loaded save file.
+- userInterface -- an interface bridging the game to an arbitrary UI to describe it.
+
+### `game.js`
+
+Describes the game loop by organizing necessary dependencies and queuing the game actions in the passed `EventBus`.
+
+We use the setup logic in `game.js` to bind out `UI` and `gameState` to all of our `Rules` (see section below).
+
+### Outputs
+
+The game loop ends produces two outputs, one of which is arguable a byproduct.
```

Comment:
Is there a word missing here? The game loop ends produces two outputs?

As in, when the game loop ends, there are two outputs?